e2e/benchmarks/byowl/base.yaml

```diff
@@ -4,11 +4,8 @@ metadata:
   name: byowl-benchmark
   namespace: my-ripsaw
 spec:
-  elasticsearch:
-    server: ES_SERVER
-    port: ES_PORT
   metadata:
-    collection: true
+    collection: false
```

Comment:
why have the base default to no ES or meta collection?

Reply:
Because if you run the tests without the proper ES environment variables set the tests will fail with metadata collection on. 

So essentially the default is no metadata collection and the testing framework will turn it on if the appropriate ES variables are given. That way the tests work with and without ES